build.gradle.kts

```diff
@@ -17,10 +17,15 @@ plugins {
     `java-library`
 }
 
+val version: String by project
+val groupId: String by project
 val swagger: String by project
 val rsApi : String by project
 
 allprojects {
+    version = version
+    group = groupId
```

Comment:
nit: group vs groupId while in the local var for version, it is the named the same